src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -826,6 +826,17 @@ if term.isColour() then
     settings.set( "bios.use_multishell", true )
     settings.set( "list.file_color", "white" )
     settings.set( "list.directory_color", "green" )
+    settings.set( "edit.backgroundColor", "black" )
```

Comment:
The reason, why this is not underscore_case, is that the programs has her config for Colours in the code. I toke the names from there.